code/english.lisp

```diff
@@ -22,6 +22,10 @@
       (let ((dictionary (reduce #'load-file '("data/english.txt"
                                               "data/english-additions.txt")
                                 :initial-value (make-instance 'dictionary))))
+        (note "Compressing dictionary")
```

Comment:
Just realized that L9's `:per-line-prefix "; "` should be `";; "` to follow CLHS 2.4.4.2.

Reply:
Done.